.github/scripts/run_e2e_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
does it works with the script `run_local_e2e_pipeline.sh`?

Reply:
It does, a simplified log:

```
     Running unittests (target/debug/deps/aleph_node-9c6f605cf1eff32e)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests (target/debug/deps/aleph_runtime-54abc09c46a98ea6)

running 1 test
test __construct_runtime_integrity_test::runtime_integrity_tests ... ok

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

     Running unittests (target/debug/deps/finality_aleph-f4bd448316b6f4fe)

running 127 tests

...

test result: ok. 127 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 53.54s

     Running unittests (target/debug/deps/pallet_aleph-621c615a61bba40d)

running 8 tests

...

test result: ok. 8 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.01s

     Running unittests (target/debug/deps/pallet_elections-6760ee537da352c7)

running 2 tests
test mock::__construct_runtime_integrity_test::runtime_integrity_tests ... ok
test tests::test_elect ... ok

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.01s

     Running unittests (target/debug/deps/primitives-27c8394dd7942b6e)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s
```

Reply:
Now it does :P